### PR TITLE
Add the packages files

### PR DESCRIPTION
**Pull request by jankatins**
Fix my mistakes... Omited the files in the last commit :-(

### CODE REVIEW

File: src/contrib/PACKAGES [hunk 1]
```diff
@@ -3,16 +3,16 @@ Version: 0.3
 Depends: R (>= 3.0.1)
 Imports: repr, base64enc
 License: MIT + file LICENSE
-MD5sum: 23d38d54c86468ef8bebfab5518620ad
```
Comment (takluyver) [inline src/contrib/PACKAGES:6]: Why is the MD5 of version 0.3 changing?

Comment (takluyver) [inline src/contrib/PACKAGES:6]: Of IRdisplay, even. And repr and rzmq also changed.

Comment (jankatins) [inline src/contrib/PACKAGES:6]: Oh my, I have to fix my scripts: The script compiles (and copies) all packages and then manually commited only the new packages.

It seems that rebuilding a package will change the content (probably a date?)... I will fix this, just a second...

**Comment by jankatins** [comment]
Fixed, sorry for this :-/

File: src/contrib/PACKAGES [hunk 2]
```diff
@@ -26,6 +26,6 @@ NeedsCompilation: no
 Package: rzmq
 Version: 0.7.7
 License: GPL-3
-MD5sum: 297240be35b02cb04ce378f31083d67d
+MD5sum: 4a0114038502b470f7e62cd48ec89934
```
Comment (takluyver) [inline src/contrib/PACKAGES:29]: This one still changed.

Comment (jankatins) [inline src/contrib/PACKAGES:29]: not sure why. I suspect that I made the same mistake during the last upload and this went in then :-/

Comment (takluyver) [inline src/contrib/PACKAGES:29]: OK, the new hash is the one that's in the repo, so I'll merge this. Thanks for fixing it quickly!

**Pull request merged**